ConsoleApp/Generics/Stack.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+namespace ConsoleApp;
+
+class Stack<T>
+{
+    private record StackItem
+    {
+        public T Value { get; init; }
+        public StackItem Previous { get; set; }
+
+    }
+    private StackItem _head;
+    public int Length { get; private set; }
+    public void Push(T element)
+    {
+        var newItem = new StackItem
+        {
+            Value = element
+
+        };
+
+        if (_head != null)
```

Comment:
no need to check for it, if you're going to do it always in case if stack empty null will be assigned